########################################
# Messages
########################################
uint8 MorA
uint8 EStop
uint8 Gear
uint8 alive
uint16 KPH
int16 Deg
uint8 brake
int32 ctrlIndex
bool aliveCtrl